types/chrome/index.d.ts

```diff
@@ -806,15 +806,15 @@ declare namespace chrome.browserAction {
      * @param tabId The id of the tab for which you want to modify the browser action.
      * @return The `enable` method provides its result via callback or returned as a `Promise` (MV3 only). It has no parameters.
      */
-    export function enable(tabId?: number): Promise<void>;
+    export function enable(tabId?: number | null): Promise<void>;
     /**
      * Since Chrome 22.
      * Enables the browser action for a tab. By default, browser actions are enabled.
      * @param tabId The id of the tab for which you want to modify the browser action.
      * @param callback Supported since Chrome 67
      */
-    export function enable(callback: () => void): void;
-    export function enable(tabId: number, callback: () => void): void;
+    export function enable(callback?: () => void): void;
+    export function enable(tabId: number | null | undefined, callback?: () => void): void;
```

Comment:
`tabId: number | null | undefined,` -> `tabId: number | null,`

Reply:
Thanks for the comment. I wanted to make sure DT permits any code which is accepted by Chrome. When I run in the console the following works:
![image](https://user-images.githubusercontent.com/45960703/223230957-17ae436e-e581-4439-9fe9-ae8a4007390b.png)